### PR TITLE
Clarify README around exists operator

### PR DESCRIPTION
**Pull request by Trevoke**

### CODE REVIEW

**Comment by robconery** [comment]
Honestly I think this is more confusing. If you're working with JSONB you should know/understand what the existence operator is - adding a sentence that says "we implemented it" implies that we somehow created an existence operator.

I know you were confused by this - and I'm OK adjusting the README, but I think this causes more confusion.

**Comment by Trevoke** [comment]
How about this wording?

**Comment by robconery** [comment]
Looks good - thanks :)
